Libraries/src/Amazon.Lambda.Annotations/LambdaFunctionAttribute.cs

```diff
@@ -8,6 +8,11 @@ namespace Amazon.Lambda.Annotations
     [AttributeUsage(AttributeTargets.Method)]
     public class LambdaFunctionAttribute : Attribute
     {
+        /// <summary>
+        /// The serializer to use.
+        /// </summary>
+        public string Serializer { get; set; }
```

Comment:
We have a mix of approaches here. For class libraries user's don't set this value but instead use the assembly attribute. But for executable they would need to set this property. This will cause class library users to think they can set this property and that will work. I think we need to resolve what is the serializer via the attributes so we have a consistent experience.

Reply:
Understood and updated to use the assembly OR the method attribute. I think this also fixes another issue that was opened yesterday #1572